libreyolo/validation/depth_validator.py

```
@@ -0,0 +1,201 @@
+            output = output[0]
+        output = torch.as_tensor(output)
+
+        targets = batch[1]
+        target_hw = tuple(targets.shape[-2:])
+        if output.ndim == 3:
+            output = output.unsqueeze(1)
+        if output.ndim != 4 or output.shape[1] != 1:
+            raise ValueError(
+                f"Depth validation expects [B, 1, H, W] output, got shape "
+                f"{tuple(output.shape)}."
+            )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)